src/somef/export/json_export.py

```diff
@@ -192,6 +210,37 @@ def format_date(date_string):
     # now, prune out the variables that are None
     save_json_output(pruned_output, outfile, None, pretty=pretty)
 
+def extract_scholarly_article_properties(bibtex_entry, scholarlyArticle):
+    """
+    Extract datePublished, issn and pagination fromg BibTeX and append to scholarlyArticle object
+    
+    Params:
+        - bibtex_entry (str): Entrada BibTeX en formato string.
+        - scholarlyArticle (dict): Diccionario donde se almacenarán los datos extraídos.
+    """
+
+    # regular expresions properties
+    issn_match = re.search(r'issn\s*=\s*{([\d-]+)}', bibtex_entry)
+    year_match = re.search(r'year\s*=\s*{(\d{4})}', bibtex_entry)
+    month_match = re.search(r'month\s*=\s*{(\d{1,2})}', bibtex_entry)
+    pages_match = re.search(r'pages\s*=\s*{([\d-]+)}', bibtex_entry)
```

Comment:
Add constants for regular expressions here: https://github.com/KnowledgeCaptureAndDiscovery/somef/blob/master/src/somef/utils/constants.py